app/src/services/storage.js

```diff
@@ -86,6 +92,34 @@ const objectStorageService = {
     return this._s3Client.send(new DeleteObjectCommand(params));
   },
 
+  /**
+   * @function deleteObjectTagging
+   * Deletes the tags of the object at `filePath`
+   * @param {string} options.filePath The filePath of the object
+   * @param {number} [options.versionId] Optional specific versionId for the object
+   */
+  async deleteObjectTagging({ filePath, tags, versionId }) {
```

Comment:
Tags isn't being used or wired into the function. Was this an oversight? If it is, we should probably set tags to be an empty set of some form for a default.